### PR TITLE
Badge component added

### PR DESCRIPTION
**Pull request by rawris**
Badge component added, currently it supports.:

1. Verified
2. Unverified

more configurations can be added by updating the javascript object inside the badge component

### CODE REVIEW

**Comment by rawris** [comment]
@Deep-Codes @karmakoder can any one review and merge this?

File: components/Badge.jsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        title: 'Not Verified',
+        className: 'text-yellow-400 w-6',
+        textClassName: 'text-lg'
+    }
+};
+
+const Badge = ({
+    badgeType,
+}) => {
+    if (!badgeType || (badgeType && !badgeData[badgeType.toLowerCase()])) {
```
Comment (Deep-Codes) [inline components/Badge.jsx:32]: Refer: https://life-bo2dzdlc7-coronasafe.vercel.app/maharashtra/nashik/medicine

Some of the Cards have no `verified` and `not verified` icon

So you gotta handle that case too. (Would Make sense right?)

**Review by Deep-Codes — CHANGES_REQUESTED** [review]

File: components/Badge.jsx (hunk 1 above)
Comment (Deep-Codes) [reply components/Badge.jsx:32]: I would make the Changes :)

Comment (rawris) [reply components/Badge.jsx:32]: I think I know how to handle this case, I just need to use || 'unverified'. I'll do it

Comment (rawris) [reply components/Badge.jsx:32]: This is done

**Review by Deep-Codes — APPROVED** [review]

File: components/Badge.jsx (hunk 1 above)
Comment (Deep-Codes) [reply components/Badge.jsx:32]: Yeah looks good 🚀 Thanks for This!

**Pull request merged**